GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -36,7 +36,6 @@ public abstract class ProductUpgrader : IDisposable
                 UpgraderToolConfigFile,
                 "GVFS.Common.dll",
                 "GVFS.Platform.Windows.dll",
-                "Microsoft.Diagnostics.Tracing.EventSource.dll",

```

Comment:
Should we also remove any references to `Microsoft.Diagnostics.Tracing` in projects\config files?

Reply:
Yes - we should. I will go and remove the other ones I find.